IP restrictions Note added (#4657)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source.md

import useBaseUrl from '@docusaurus/useBaseUrl';

:::note
- Collecting data from Azure Event Hubs using this Cloud-to-Cloud collection method has a supported throughput limit of 1MB/S (86GB/day) for a named Event Hub egress rate. We recommend using the [Azure Event Hubs Source for Logs ](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) if you require higher throughput. 

> - The only caveat is this Cloud-to-Cloud collection method supports IP restrictions and the [Azure Event Hubs Source for Logs ](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) does not. If you require higher throughput and have IP address restrictions on Event Hubs, then we recommend splitting your Event Hubs into smaller namespaces keeping within the 1MB/S (86GB/day) limit and create this Cloud-to-Cloud collection method for each namespace.

Reviewer: ```suggestion
- The only caveat is this Cloud-to-Cloud collection method supports IP restrictions and the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) does not. If you require higher throughput and have IP address restrictions on Event Hubs, then we recommend splitting your Event Hubs into smaller namespaces keeping within the 1MB/S (86GB/day) limit and create this Cloud-to-Cloud collection method for each namespace.
```

> - Collecting data from Azure Event Hubs using this Cloud-to-Cloud collection method has a supported throughput limit of 1MB/S (86GB/day) for a named Event Hub egress rate. We recommend using the [Azure Event Hubs Source for Logs ](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) if you require higher throughput. 
> - The only caveat is this Cloud-to-Cloud collection method supports IP restrictions and the [Azure Event Hubs Source for Logs ](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) does not. If you require higher throughput and have IP address restrictions on Event Hubs, then we recommend splitting your Event Hubs into smaller namespaces keeping within the 1MB/S (86GB/day) limit and create this Cloud-to-Cloud collection method for each namespace.

Reviewer: ```suggestion
- Collecting data from Azure Event Hubs using this Cloud-to-Cloud collection method supports a throughput limit of 1MB/s (86GB/day)  per named Event Hub egress rate. If you require higher throughput, we recommend using the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source).
- The only caveat is this Cloud-to-Cloud collection method supports IP restrictions and the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/) does not. If you require higher throughput and have IP address restrictions on Event Hubs, consider splitting your Event Hubs into smaller namespaces, each staying within the 1MB/s (86GB/day) limit, and create a Cloud-to-Cloud collection method for each namespace.
```
